### PR TITLE
MozCoffee 7 Blog added

### PR DESCRIPTION
**Pull request by manrajgrover**
MozCoffee 7 Blog added. Let me know if image added is correct.

### CODE REVIEW

File: content/mozcoffee-7-blog.rst [hunk 1]
```diff
@@ -0,0 +1,42 @@
+MozCoffee Delhi 7.0
+############################################
```
Comment (CuriousLearner) [inline content/mozcoffee-7-blog.rst]: You need as many `#` as long the title is. So please remove extra hashes.

Comment (manrajgrover) [inline content/mozcoffee-7-blog.rst]: Let me know if image added is correctly linked

Comment (CuriousLearner) [inline content/mozcoffee-7-blog.rst]: I would be only able to check if it's working in night.

**Comment by CuriousLearner** [comment]
Also for content review @anupkumarmishra @bhuvneshdogra @Slayslot @Mozpacers/contributors 

Let me know if it is good to go content wise.

**Comment by manrajgrover** [comment]
@CuriousLearner Let's land this. Let me know if any other change is required.

**Comment by manrajgrover** [comment]
@CuriousLearner Please test this and lets merge this ASAP!

**Comment by CuriousLearner** [comment]
Hi @ManrajGrover 

Sorry for taking this too long. Your image is not visible on the post. Can you link it properly with Markdown?

**Comment by CuriousLearner** [comment]
ahh, @ManrajGrover This is an RST file, why are you even using markdown?

**Comment by manrajgrover** [comment]
@CuriousLearner Let me know if it is fixed

File: content/mozcoffee-7-blog.rst [hunk 2]
```diff
@@ -0,0 +1,44 @@
+MozCoffee Delhi 7.0
+###################
+:date: 2016-08-2 18:06
+:author: Manraj Singh
+:category: blog
```
Comment (CuriousLearner) [inline content/mozcoffee-7-blog.rst]: Please change category to mozcoffee

Comment (manrajgrover) [reply content/mozcoffee-7-blog.rst]: Fixed

**Comment by CuriousLearner** [comment]
There are still some issues, but I'm merging this and will correct those later.

**Comment by manrajgrover** [comment]
@CuriousLearner Let me know, I'll fix them. Also, we should add how to test blog locally to README. It will make reviewing work easy and fast.

**Comment by CuriousLearner** [comment]
@ManrajGrover Sure, I'm creating issues :)

If you have time, fix them :)

**Comment by manrajgrover** [comment]
@CuriousLearner Sure! Let me know when it is deployed :)

**Pull request merged**